app/domain/WalletTransaction.js

```diff
@@ -5,37 +5,29 @@ import type { AssuranceMode, AssuranceLevel } from '../types/transactionAssuranc
 import { assuranceLevels } from '../config/transactionAssuranceConfig';
 
 export type TrasactionAddresses = { from: Array<string>, to: Array<string> };
-export type TransactionState = 'pending' | 'failed' | 'ok';
-export type TransactionType = 'card' | 'expend' | 'income' | 'exchange' | 'self' | 'multi';
 
-export const transactionStates: {
-  PENDING: TransactionState,
-  FAILED: TransactionState,
-  OK: TransactionState,
-} = {
+export const transactionStates = Object.freeze({
```

Comment:
maybe we should enable `'use strict';` for this 🤔 at least for dev mode

Reply:
@SebastienGllmt in practice if you try to change a object that was created using Object Freeze, nothing will happen, but if strict mode is enabled it will throw an error, which I think it will be good to let us know that we are doing something wrong

Reply:
Try the playground here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze and add the strict mode to see how is different

Reply:
Javascript modules are strict by default (and in fact I don't think you can make them non-strict) so we already have this behavior

Reply:
http://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code